specification/postgresql/resource-manager/readme.go.md

```diff
@@ -28,6 +28,7 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-flexibleserver-2022-12-01
```

Comment:
Looks the configure file for Go is not correct, could you follow https://github.com/Azure/azure-rest-api-specs/blob/main/specification/agrifood/resource-manager/readme.go.md to update this file?

Reply:
@ArcturusZhang can you please check if this is really a breaking change. If yes, please suggest what needs to be updated here as only new tag has been added in this change
![image](https://user-images.githubusercontent.com/61298407/207540786-45ed4173-b057-42fc-b461-ce82faa14065.png)